### PR TITLE
Updates to DEDIS

### PR DESCRIPTION
**Pull request by ineiti**
By request of Pierluca, updates to the DEDIS repo.

### CODE REVIEW

File: data/DEDIS/projects.yaml [hunk 1]
```diff
@@ -186,9 +247,13 @@ projects:
...
     name: E-voting
-    description: E-voting using a blockchain
+    description: Decentralized, encrypted, fully anonymous e-voting
+    tech_desc: >
+      Our e-voting system is inspired by the first version of Helios where the encrypted ballots are shuffled and
+      anonymized before they are decrypted. Instead of the shuffle used in Helios, we implemented a Neff shuffle
+      which is much faster than the original Helios shuffle.
     layman_desc: >
-      The e-voting used in the EPFL voting in summer 2018. It allows users
+      The e-voting used in the EPFL voting in summer 2018 and 2020. It allows users
```
Comment (cgrigis) [inline data/DEDIS/projects.yaml:256]: since 2018?

**Review by cgrigis — CHANGES_REQUESTED** [review]
Minor points, otherwise LGTM

**Review by cgrigis — APPROVED** [review]

**Comment by ineiti** [comment]
should've put this in Draft mode, sorry, and not asked for your review. Was not ready for merge...

**Comment by cgrigis** [comment]
Oops, sorry. Want me to revert?

**Comment by ineiti** [comment]
No revert necessary, I hope DEDIS will not complain ;)

Just refining the workflow a bit 
- @ineiti needs to actually mark PRs as drafts (started with #172)
- generally: only merge PRs that are in the `Ready4Merge` pipeline - not sure if we can enforce that with github

**Pull request merged**